packages/builder/uni-builder/package.json

```diff
@@ -73,6 +73,7 @@
     "webpack-subresource-integrity": "5.1.0"
   },
   "devDependencies": {
+    "@modern-js/builder-plugin-node-polyfill": "workspace:*",
```

Comment:
this package will be removed soon, so we should not depend on it

Reply:
This will be removed along with other builder dependencies in modern.js repo.